CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md

```diff
@@ -193,3 +193,26 @@ You can show a bottom navigation bar by passing true to `GiniConfiguration.share
 your own by implementing the `HelpBottomNavigationBarAdapter` interface and passing it to `GiniConfiguration.shared.helpNavigationBarBottomAdapter`.
 
 You can find more details [here][https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/3.0.0-beta01/features.html#help-screen-customization] and [here](https://www.figma.com/file/H4WFJ4xaw4YNU4VaJYWiQq/iOS-Gini-Capture-SDK-2.0.0-UI-Customisation?node-id=40%3A584&t=LCqm8Cwfh9PGUM0E-1).
+
+## Analysis screen
+
+The new analysis screen uses the global UI customization options. You can discard the old screen specific
+customizations.
+
+The following string keys now represent suggestion titles with new keys added for describing the tips.
```

Comment:
[Suggestion] could you please move it to the breaking changes? 🙈

Reply:
I did not consider it. breaking change, that's why I left it there, but it is safe now 💪 😄